db.go

```diff
@@ -67,6 +81,18 @@ const emptyReputationTableSQL = `
 TRUNCATE TABLE reputation;
 `
 
+const createViolationReputationWeightsTableSQL = `
+CREATE TABLE IF NOT EXISTS violation_reputation_weights (
+violation_type varchar(128) PRIMARY KEY NOT NULL,
+reputation int NOT NULL CHECK (reputation >= 0 AND reputation <= 100),
```

Comment:
Currently a violation sets a reputation directly instead of being added to it or subtracted from it.

I'm not sure if that's what we want.